### ISSUE TITLE
Area page redesign

### ISSUE THREAD

**Issue by gbp** [mysociety/stop-and-search#2]
Most of the current content can be dropped in its entirety.

The area page for the stop and search prototype should show data from either a single council or a single police force.

It will display the difference between different ethnicity (Black or White) in stop and search data.

The main data points are:
1. stop rate vs non-stop rate out of a total population
2. the odds ratio comparing the different ethnicities
3. the risk ratio comparing the different ethnicities

For all of these we have data spanning multiple years so seeing trends over time would be a good way of virtualising the data.

There will also be different metrics which the area should be filterable by, these include 1. object of the search, 2 legislation 3. outcome. Each of these will have different values which can be selected. I think we should allow users to add multiple metrics to be visible per page.

I have attempted to mock up what I have be visualising in my head, I hope it makes sense:

![area-page](https://github.com/mysociety/stop-and-search/assets/5426/852ff169-a937-4790-8467-5ddbf054d7e2)

![area-page-charts](https://github.com/mysociety/stop-and-search/assets/5426/310355d0-3bd7-4a9a-976c-4be3ced459da)

**Comment by lucascumsille** [comment]
Some thing for further discussion and might not be related to this page, but I'll use this ticket anyway:

Not sure if it's part of the current scope, but it seems natural for me that if we are delivering data about areas/districts. Should we also have a page with no map(due to space), where you can just play with the data, no by area, but in a national level.

A few ideas from the top of my mind. We have this 3 ratios at the moment:

- stop rate vs non-stop rate out of a total population
- the odds ratio
- the risk ratio 

Basically same info as the current area page, but in a national/macro level.

I would be interested in checking more than my council area. For example: Is there a difference between countries(Example Scotland v/s UK)? 
- Checking national data between years.
- Checking gender by national.

**Comment by JenMysoc** [comment]
Since we've done the show and tell and are moving on to the next iteration, do we still need this open? Can we open a new tticket with the info for the next iteration based on feedback from the show and tell?

**Comment by lucascumsille** [comment]
@JenMysoc @gbp 
that's fine with me =)
